lib/cli/test/unit/Cardano/CLISpec.hs

```diff
@@ -384,9 +454,24 @@ spec = do
             , "  EPOCH_NUMBER             epoch number parameter or 'latest'"
             ]
 
+        ["key", "root", "--help"] `shouldShowUsage`
+            [ "Usage:  key root --type KEYTYPE MNEMONIC_WORDS..."
```

Comment:
I am going to be slightly pedantic here but `MNEMONIC_WORDS` should be singular. The meta-type is repeated. Also, why is there an underscore between `MNEMONIC` and `WORDS` but there's none between `KEY` and `TYPE`? The underscore reads better, but it should be used consistently across the interface.

Reply:
Done. `MNEMONIC_WORD...` and `KEY_TYPE`.

(Underscores are already used elsewhere in the CLI)